lib/Settings/Personal.php

```diff
@@ -82,7 +64,16 @@ public function displayPanel() {
 
 		return new TemplateResponse('ransomware_protection', 'personal', [
 			'disabledUntil'			=> $disabledUntil,
-		], '');
+		], TemplateResponse::RENDER_AS_BLANK);
+	}
+
+	public function getSection(): string {
+		// FIXME after https://github.com/nextcloud/server/pull/22589 is merged
+		// return 'additional';
+		return 'personal-info';
```

Comment:
Either this should be `'additional'` or we use a very low priority on `personal-info` so it's at the end there.

https://github.com/nextcloud/server/pull/22589